ibm/service/satellite/data_source_ibm_satellite_host_script.go

```diff
@@ -155,6 +160,12 @@ func dataSourceIBMSatelliteAttachHostScriptRead(d *schema.ResourceData, meta int
 		scriptPath = filepath.Join(scriptDir, "addHost.sh")
 	}
 
+	//If the user supplied link agent endpoint, use reduced firewall attach script
+	if _, ok := d.GetOk("host_link_agent_endpoint"); ok {
+		host_link_agent_endpoint := d.Get("host_link_agent_endpoint").(string)
```

Comment:
this can be simplified a bit. what do you think about this one?
```suggestion
	if hlae, ok := d.GetOk("host_link_agent_endpoint"); ok {
		host_link_agent_endpoint := hlae.(string)
```

Reply:
Yeah this sounds good. I will make the change shortly